src/release.js

```diff
@@ -119,6 +127,23 @@ module.exports = async function ({ github, context, inputs }) {
       inputs
     )
 
+    try {
+      // post a comment about release on npm to any linked issues in the
+      // any of the PRs in this release
+      await notifyIssues(
+        github,
+        release.body,
+        packageVersion, // npm version format ie no 'v' in front of version
+        owner,
+        repo,
+        release.html_url,
```

Comment:
this function is already getting many arguments. let's maybe provide the whole release in a single one instead of multiple. the release will probably also contain information about the repo and the owner I suppose

Reply:
The requested refactoring is done.